x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/right/components/highlighted_fields_cell.tsx

```diff
@@ -34,6 +33,11 @@ export interface HighlightedFieldsCellProps {
    * Highlighted field's value to display
    */
   values: string[] | null | undefined;
+  /**
+   * Maintain backwards compatibility // TODO remove when possible
+   * Only needed if alerts page flyout (which has PreviewLink), NOT in the AI for SOC alert summary flyout.
+   */
+  scopeId?: string;
```

Comment:
i think it doesn't hurt to have a `scopeId` for AI4SOC (just literally `ai-for-soc`?) so we don't have to do a clean up later.

as discussed, maybe a explicit prop called `disablePreview` is better than checking `scopeId` so that we can keep the logic separate

Reply:
Yup I actually reverted that change locally. Will push the commit soon, finishing unit tests!

Reply:
I decided to keep it optional here, because having `scopeId={'ai-for-soc'}` in the parent component was really weird. It was confusing to me to put a value that wouldn't be used, and I think a few months from now we would question again why having a value there, knowing that it's not used anywhere...